### ISSUE TITLE
Fix alt text for vignette images

### ISSUE THREAD

**Issue by nanxstats** [nanxstats/protr#46]
Running pkgdown now hints missing alt text issues for vignette images. Should fix this by using `knitr::include_graphics()` chunks instead of writing HTML manually.
